Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/HOKSAMLAssertionBuilder.java

```diff
@@ -751,21 +739,46 @@ protected boolean isConditionsDefaultValueEnabled() {
         return Boolean.TRUE;
     }
 
+    public static String issuerStringfromList() {
+        List<Object> oIssuer = Collections.emptyList();
+        String sIssuer;
+        try {
+            oIssuer = PropertyAccessor.getInstance().getPropertyList(PROPERTY_FILE_NAME, PROPERTY_SAML_ISSUER_NAME);
+        } catch (PropertyAccessException ex) {
+            LOG.error("HOKSAMLAssertionBuilder can not access assertioninfo property file: {}",
+                ex.getLocalizedMessage(), ex);
+        }
+        int iSize = oIssuer.size();
+        if (oIssuer.isEmpty() || iSize != 6) {
```

Comment:
You can read comma separated string by providing escape character. Simply put the property as:
SamlIssuerName=CN=SAML User\,OU=SU\,O=SAML User\,L=Los Angeles\,ST=CA\,C=US 

Also, add a testcase in ProperyDaoTest class to test this.